app/core/src/main/java/com/fsck/k9/preferences/SettingsImporter.java

```diff
@@ -157,6 +160,8 @@ public static ImportContents getImportStreamContents(InputStream inputStream)
      *         same UUID is found in the settings file.<br>
      *         <strong>Note:</strong> This can have side-effects we currently don't handle, e.g.
      *         changing the account type from IMAP to POP3. So don't use this for now!
+     * @param outboxName
```

Comment:
Passing a localized outbox name feels wrong. Since we never use it anyway, I'd say we use a constant `Account.OUTBOX_NAME` with the value "Outbox".

Reply:
Done